task/controller/user.js

```diff
@@ -1,37 +1,92 @@
 const Login = require("../models/user");
 const jwt = require("jsonwebtoken");
-var secret = "login";
+const path = require("path");
+const fs = require("fs");
+const csv = require("csv-parser");
+const { infoLogger, errorLogger } = require("../../logger");
+const { user } = require("./notes");
+const { insertMany, remove } = require("../models/user");
+require("dotenv").config();
+const secret = process.env.SECRET;
+const employee = require("../models/employee");
+let __basedir = path.resolve();
+
 exports.register = async (req, res) => {
   try {
-    let user = await Login.create(req.body);
+    let user = await Login.create(req.body, req.files);
+    infoLogger.info(user);
+    res
+      .status(200)
+      .json({ success: true, message: "register successfully", data: user });
+  } catch (error) {
+    errorLogger.error(error.message);
+    res.status(400).json({ success: false, message: error.message });
+  }
+};
+exports.name = async (req, res) => {
+  try {
+    let Tutorial = [];
+    let path = __basedir + "/file/" + req.file.filename;
+    // console.log("path", path);
+    fs.createReadStream(path)
+      .pipe(csv())
+      .on("error", (error) => {
+        throw error.message;
+      })
+      .on("data", (row) => {
+        Tutorial.push(row);
+      })
+      .on("end", () => {
+        // console.log("todos", Tutorial);
+        employee.insertMany(Tutorial);
+        fs.unlinkSync(path);
+      });
 
-    res.json({ success: true, message: "register successfully", data: user });
-  } catch (err) {
-    res.json({ success: false, message: err.message });
+    let data = new Login();
```

Comment:
Why registration code is into this API?
CSV file upload code must be a different API.